.codecov.yaml

```diff
@@ -0,0 +1,5 @@
+ignore:
+  - "**/zz_*_generated.go" # Ignore generated files.
```

Comment:
This is wrong, see: https://codecov.io/gh/knative-sandbox/discovery/pull/60/tree?path=pkg%2Fapis%2Fdiscovery%2Fv1alpha1